R/read_from_dhis2-helpers.R

```diff
@@ -166,3 +168,40 @@ dhis2_get_attributes_from_user <- function(args_list) {
     end_date           = end_date
   )
 }
+
+#' Make request query parameters
+#'
+#' @param dataset a vector or a list of comma-separated data set identifiers
+#' @param org_unit a vector or a list of comma-separated organisation
+#'    unit identifiers
```

Comment:
If these are actually vectors (which I agree they should be), it should be necessary to use `strsplit()` and this entire function shouldn't be necessary. We can pass params as a list directly to httr2